docs/fundamentals/aspire-sdk-templates.md

```diff
@@ -0,0 +1,165 @@
+---
+title: .NET Aspire templates
+description: Learn how to install the .NET Aspire templates, and how to use them to create new apps.
+ms.date: 11/04/2024
+zone_pivot_groups: dev-environment
+uid: dotnet/aspire/templates
+---
+
+# .NET Aspire templates
+
+There are a number of .NET Aspire project templates available to you. You can use these templates to create full .NET Aspire solutions, or add individual projects to existing .NET Aspire solutions.
+
+The .NET Aspire templates are available in the [📦 Aspire.ProjectTemplates](https://www.nuget.org/packages/Aspire.ProjectTemplates) NuGet package.
+
+## Install the .NET Aspire templates
+
+:::zone pivot="visual-studio"
+
+The .NET Aspire templates are installed automatically when you install Visual Studio 17.9 or later.
+
+:::zone-end
+:::zone pivot="vscode,dotnet-cli"
+
+To install these templates, use the [dotnet new install](/dotnet/core/tools/dotnet-new-install) command, passing in the `Aspire.ProjectTemplates` NuGet identifier.
+
+```dotnetcli
+dotnet new install Aspire.ProjectTemplates
+```
+
+:::zone-end
+
+## List the .NET Aspire templates
+
+:::zone pivot="visual-studio"
+
+The .NET Aspire templates are installed automatically when you install Visual Studio 17.9 or later. To see what .NET Aspire templates are available, select **File** > **New** > **Project** in Visual Studio, and search for "Aspire" in the search bar (<kbd>Alt</kbd>+<kbd>S</kbd>). You'll see a list of available .NET Aspire project templates:
```

Comment:
Let me know if you want more details or if you want me to suggest changes here that represent the ways to get the templates, and you can review.

Reply:
Yes, please... Please make suggestions here in the PR, so that I can review/edit and accept.

Reply:
```suggestion
The .NET Aspire templates are installed automatically when you install Visual Studio 17.12 or later. To see what .NET Aspire templates are available, select **File** > **New** > **Project** in Visual Studio, and search for "Aspire" in the search bar (<kbd>Alt</kbd>+<kbd>S</kbd>). You'll see a list of available .NET Aspire project templates:
```